src/components/UserInfo/user_info.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { User } from '../Type/User';
+
+// import '../../App.scss';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <>
+      <div className="list_item">
+        {user ? user.username : null}
+      </div>
```

Comment:
```suggestion
      {user && (
        <div className="list_item">
          {user.username}
        </div>
      )}
```
Let's use conditional rendering